ci/tests/e2e-go-scaffold.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -ex
+
+# Scaffold project (image name and local image args don't matter at this stage as those only affect the operator manifest)
+source ./hack/tests/scaffolding/e2e-go-scaffold.sh
+
+pushd $BASEPROJECTDIR/memcached-operator
+go build -gcflags "all=-trimpath=${GOPATH}" -asmflags "all=-trimpath=${GOPATH}" -o /memcached-operator $BASEPROJECTDIR/memcached-operator/cmd/manager
```

Comment:
I suppose assuming `GOPATH` is always set in OpenShift CI.

Reply:
Yes. It may also be reasonable to trim `$BASEPROJECTDIR` since that's where this project is, but trimming multiple build paths is only being added in 1.13 so I guess we can't do that yet ¯\\_(ツ)_/¯

This is only an internal image used for testing though, so it shouldn't matter